optimize/upgrade/src/test/java/io/camunda/optimize/upgrade/AbstractUpgradeIT.java.out

```diff
@@ -6,348 +6,348 @@
-//         elasticConfig.getHost(),
-//         elasticConfig.getHttpPort(),
-//         IntegrationTestConfigurationUtil.getDatabaseMockServerPort());
-//   }
-// 
-//   private String getLogIndexAlias() {
-//     return indexNameService.getOptimizeIndexAliasForIndex(
-//         DatabaseConstants.UPDATE_LOG_ENTRY_INDEX_NAME);
-//   }
-// 
-//   private Settings createIndexSettings(final IndexMappingCreator indexMappingCreator) {
-//     try {
-//       return ElasticSearchIndexSettingsBuilder.buildAllSettings(
-//           configurationService, indexMappingCreator);
-//     } catch (final IOException e) {
-//       throw new OptimizeRuntimeException("Could not create index settings");
-//     }
-//   }
-// 
-//   protected void insertTestDocuments(final int amount) throws IOException {
-//     final String indexName = TEST_INDEX_V1.getIndexName();
-//     final BulkRequest bulkRequest = new BulkRequest();
-//     for (int i = 0; i < amount; i++) {
-//       bulkRequest.add(
-//           new IndexRequest(indexName)
-//               .source(
-//                   String.format("{\"password\" : \"admin\",\"username\" : \"admin%d\"}", i),
-//                   XContentType.JSON));
-//     }
-//     prefixAwareClient.bulk(bulkRequest);
-//     prefixAwareClient.refresh(new RefreshRequest(indexName));
-//   }
-// 
-//   public void deleteAllDocsInIndex(final IndexMappingCreator index) {
-//     final DeleteByQueryRequest request =
-//         new DeleteByQueryRequest(indexNameService.getOptimizeIndexAliasForIndex(index))
-//             .setQuery(matchAllQuery())
-//             .setRefresh(true);
-// 
-//     try {
-//       prefixAwareClient
-//           .getHighLevelClient()
-//           .deleteByQuery(request, prefixAwareClient.requestOptions());
-//     } catch (final IOException | ElasticsearchStatusException e) {
-//       throw new OptimizeIntegrationTestException(
-//           "Could not delete data in index " + indexNameService.getOptimizeIndexAliasForIndex(index),
-//           e);
-//     }
-//   }
-// }
+//package io.camunda.optimize.upgrade;
+//
+//import static io.camunda.optimize.service.util.configuration.ConfigurationServiceBuilder.createDefaultConfiguration;
+//import static io.camunda.optimize.upgrade.EnvironmentConfigUtil.createEmptyEnvConfig;
+//import static io.camunda.optimize.upgrade.EnvironmentConfigUtil.deleteEnvConfig;
+//import static io.camunda.optimize.upgrade.es.SchemaUpgradeClientFactory.createSchemaUpgradeClient;
+//import static jakarta.ws.rs.HttpMethod.DELETE;
+//import static jakarta.ws.rs.HttpMethod.POST;
+//import static org.elasticsearch.index.query.QueryBuilders.matchAllQuery;
+//import static org.elasticsearch.index.query.QueryBuilders.termQuery;
+//import static org.mockserver.model.HttpRequest.request;
+//
+//import com.fasterxml.jackson.databind.ObjectMapper;
+//import io.camunda.optimize.exception.OptimizeIntegrationTestException;
+//import io.camunda.optimize.service.db.DatabaseConstants;
```

Comment:
❓ Do we need the class outcommented? You can also find this via git history no need to keep such files. :)

Reply:
Yeah, we've got a whole bunch of IT in Optimize that are currently C7 only but which we are planning to migrate to C8. We will be working our way through all of these commented out files and add them back in as shiny new C8 IT.